pyriemann_qiskit/pipelines.py

```diff
@@ -472,7 +472,7 @@ def __init__(
 
     def _create_pipe(self):
         clf_mean_logeuclid_dist_cpm = QuantumMDMWithRiemannianPipeline(
```

Comment:
This VotingClassifier should be replaced by a voting classifier Q-MDM/NHC or Q-MDM/MDM